README.md

```diff
@@ -42,6 +42,21 @@ On releases v0.8.4 and above, kubemacpool is set to apply on pods/vms that resid
 - `mutatepods.kubemacpool.io=allocateForAll` - to opt in pods mac allocation in your namespace
 - `mutatevirtualmachines.kubemacpool.io=allocateForAll` - to opt in vms mac allocation in your namespace
 
+#### How to enable/disable kubemacpool for a namespace 
+
+Kubemacpool is disabled by default on a new namespace.
+To enable kubemacpool on a specific namespace:
+```bash
+kubectl label namespace user-namespace-opting-in-pods-vms mutatepods.kubemacpool.io=allocateForAll mutatevirtualmachines.kubemacpool.io=allocateForAll
+namespace/user-namespace-opting-in-pods-vms labeled
+```
+
+To disable kubemacpool in a namespace:
+```bash
+kubectl label namespace user-namespace-opting-in-pods-vms --overwrite mutatepods.kubemacpool.io=disable mutatevirtualmachines.kubemacpool.io=disable
```

Comment:
@phoracek another option is to remove the label entirely. 
I think maybe removing is better - setting the label value to disable should be an option for the administrator to set in the mutatingWebhookConfiguration, should we want to disable it from all the namespaces. 
There's kind of a flexibility here.. what do you think?

Reply:
In this simple doc, yeah. We can just show how to drop the label.

However, please add a `> **Note**:` explaining that if "enabled" is the default, it can be opted-out by setting "disable".

Does it sound fair?

Reply:
ack